src/components/loginSignup/login.jsx

```diff
@@ -1,9 +1,12 @@
 import React, { Component } from 'react'
-import { Form, Button } from 'react-bootstrap'
+import { Form } from 'react-bootstrap'
```

Comment:
Why bootstrap? 
I thought we are moving away from it

Reply:
removed